package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Did you mean to add this file?

Reply:
Oops - thanks for spotting. It must have been in there before I started.